x-pack/solutions/search/plugins/enterprise_search/server/index.ts

```diff
@@ -44,7 +45,46 @@ export const configSchema = schema.object({
 export type ConfigType = TypeOf<typeof configSchema>;
 
 export const config: PluginConfigDescriptor<ConfigType> = {
-  deprecations: ({ unused }) => [unused('canDeployEntSearch', { level: 'warning' })],
+  deprecations: ({ deprecate, unused }) => [
+    unused('canDeployEntSearch', { level: 'warning' }),
+    (deprecationConfig, fromPath, addDeprecation, context) => {
+      [
+        deprecate('host', '9.0.0', {
+          documentationUrl: context.docLinks.enterpriseSearch.upgrade9x,
+          level: 'critical',
+          title: i18n.translate('xpack.enterpriseSearch.deprecations.config.hostTitle', {
+            defaultMessage: 'Enterprise Search host(s) must be removed',
+          }),
+        }),
```

Comment:
We could remove this one since the feature deprecation will be logged if this value is found so it will be shown twice (see screenshot). @markjhoy ?

Reply:
Intuitively, I _think_ so? But @seanstory might know the ramifications better as he added the original deprecation...

Reply:
Does the feature deprecation create a "critical" level? We want to block your upgrade until Enterprise Search has been removed from your deployment.

Reply:
yes it does.